modules/oe_content_entity/tests/src/FunctionalJavascript/StakeholderUiTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_content_entity\FunctionalJavascript;
+
+use Drupal\media\Entity\Media;
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Test Stakeholder bundle.
+ */
+class StakeholderUiTest extends BrowserTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'oe_content',
+    'oe_content_entity',
+    'oe_content_entity_contact',
+    'oe_content_entity_organisation',
+    'oe_content_entity_venue',
+  ];
+
+  /**
+   * Tests Stakeholder creation.
+   */
+  public function testStakeholderBundleUi(): void {
+    // Create dummy media entity image.
+    $file_data = file_get_contents('https://upload.wikimedia.org/wikipedia/commons/4/45/Eopsaltria_australis_-_Mogo_Campground.jpg');
+    $file = file_save_data($file_data);
+    $media = Media::create([
+      'bundle' => 'image',
+      'uid' => 1,
+      'oe_media_image' => [
+        'target_id' => (int) $file->id(),
+        'alt' => 'A nice bird',
+        'title' => 'A nice bird',
+      ],
+    ]);
+    $media->setName('Bird')->setPublished()->save();
+
+    $user = $this->drupalCreateUser([
+      'manage corporate content entities',
+      'access administration pages',
+      'create oe_organisation oe_stakeholder corporate entity',
+      'edit oe_organisation oe_stakeholder corporate entity',
+      'access oe_organisation canonical page',
+    ]);
+
+    $this->drupalLogin($user);
+
+    $this->drupalGet("/admin/content/oe_organisation/add/oe_stakeholder");
+
+    $this->getSession()->getPage()->fillField('Name', "My stakeholder");
+    $acronym_value = "My Acronym";
```

Comment:
create a assoc array with fieldname as key and corresponding values, then loop through